### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by haaakon**
Since String enum is not available in obj-c the different severitys is split into different methods

### CODE REVIEW

File: Digipost.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -906,6 +906,11 @@
 		ECF06D6019239537003BD200 /* UIViewController+Additions.m in Sources */ = {isa = PBXBuildFile; fileRef = ECF06D5F19239537003BD200 /* UIViewController+Additions.m */; };
 		ECF06D65192395A5003BD200 /* UIRefreshControl+Additions.m in Sources */ = {isa = PBXBuildFile; fileRef = ECF06D64192395A5003BD200 /* UIRefreshControl+Additions.m */; };
 		ECF06D731923A637003BD200 /* POSReceiptsTableViewDataSource.m in Sources */ = {isa = PBXBuildFile; fileRef = ECF06D721923A637003BD200 /* POSReceiptsTableViewDataSource.m */; };
+		ECF1C05D1B1460AF00DCB8A5 /* Logger.swift in Sources */ = {isa = PBXBuildFile; fileRef = ECF1C05C1B1460AF00DCB8A5 /* Logger.swift */; };
```
Comment (lillesand) [inline Digipost.xcodeproj/project.pbxproj:909]: This entire file is :hankey:

File: Digipost/APIClient.swift [hunk 2]
```diff
@@ -417,6 +417,14 @@ class APIClient : NSObject, NSURLSessionTaskDelegate, NSURLSessionDelegate, NSUR
         self.isUploadingFile = true
     }
 
+    func postLog(#uri: String, parameters: [String : AnyObject]) {
+        self.urlSessionTask(httpMethod.post, url: uri, parameters: parameters, success: { () -> Void in
+            println("success")
```
Comment (lillesand) [inline Digipost/APIClient.swift]: What happens to stdout on iOS?

File: Digipost/Logger.swift [hunk 3]
```diff
@@ -0,0 +1,55 @@
+//
+//  Logger.swift
```
Comment (lillesand) [inline Digipost/Logger.swift:2]: Can you turn off these headers in the editor? No need to tell ourselves which file we're in

File: Digipost/APIClient.swift (hunk 2 above)
Comment (lillesand) [inline Digipost/APIClient.swift]: We should always print logmessages to stdout for easier local debugging.

File: Digipost/Logger.swift [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    class func dpostLogWarn(message: String) {
+        dpostLog(DpostLogSeverity.Warn, message: message)
+    }
+
+    class func dpostLogError(message: String) {
+        dpostLog(DpostLogSeverity.Error, message: message)
+    }
+
+    private class func dpostLog(severity: DpostLogSeverity, message: String) {
+        if Logger.Constants.currentlyEnabled {
```
Comment (lillesand) [inline Digipost/Logger.swift:49]: Write to stdout as well here

File: Digipost.xcodeproj/project.pbxproj (hunk 1 above)
Comment (haaakon) [inline Digipost.xcodeproj/project.pbxproj:909]: Welcome to iOS :)

File: Digipost/Logger.swift (hunk 4 above)
Comment (haaakon) [inline Digipost/Logger.swift:49]: commit: 0cc24dee642f3ac90601356f8b7e401132a539e2

**Pull request merged**